docs/sources/reference/commandline/cli.md

```diff
@@ -1681,6 +1683,13 @@ will automatically create this directory on the host for you. In the
 example above, Docker will create the `/doesnt/exist`
 folder before starting your container.
 
+    $ sudo docker run -v /icanwrite --readonly busybox touch /icanwrite here
+
+Volumes can be used in combination with `--readonly` to control where 
+a container writes files.  The `--readonly` flag mounts the container's root
+filesystem as read only prohibiting writes to locations other than the
+specified volumes for the container.
+
```

Comment:
@fredlf ! ! ! ^^^ I'm trying to apply what you told us but @SvenDowideit is saying you are wrong or I did it wrong.